controlplane/kubeadm/internal/controllers/controller.go

```diff
@@ -595,11 +600,19 @@ func (r *KubeadmControlPlaneReconciler) reconcileDelete(ctx context.Context, con
 			"Failed to delete control plane Machines for cluster %s control plane: %v", klog.KObj(controlPlane.Cluster), err)
 		return ctrl.Result{}, err
 	}
+
+	log.Info("Waiting for control plane Machines to go away")
```

Comment:
i'd replace 'to go away' with something more formal e.g. 'to be deleted'

Reply:
But what does "to be deleted" mean? Setting the deeltionTimestamp? It's already on there :)

Reply:
IMO, 'to be deleted' != 'to have deletionTimestamp applied'.
delete is the HTTP method and kubectl call, etc.

Reply:
I'm not sure. The most precise way to say this is something like "Waiting for control plane Machines to not exist anymore"

Reply:
> "Waiting for control plane Machines to not exist anymore"

+1, or anything formal.